posthog-rails/lib/posthog/rails/railtie.rb

```diff
@@ -0,0 +1,214 @@
+# frozen_string_literal: true
+
+module PostHog
+  module Rails
+    class Railtie < ::Rails::Railtie
+      # Add PostHog module methods for accessing Rails-specific client
+      initializer 'posthog.set_configs' do |_app|
+        PostHog.class_eval do
```

Comment:
As mentioned above, let's configure the config for Rails inside `PostHog::Rails.config` instead - if possible, of course